scripts/sample-data.js

```diff
@@ -0,0 +1,145 @@
+const chalk = require('chalk')
+const urllib = require('url')
+const fs = require('fs')
+const path = require('path')
+const Web3 = require('web3')
+const Origin = require('../dist/index.js')
+
+
+// This script creates randomly generated sample listings for testing purposes.
+// Currently only written to run against local setup.
+
+// 1) Create a couple of user accounts
+// 2) Create a bunch of listings
+// 3) Users make offers on other's listings
+
+// Local Usage:
+// scripts/sampleData.js <number of listings> <number of offers>
+// docker exec -ti origin-js node scripts/sampleData.js 42 17
+
+const NUMBER_OF_LISTINGS = parseInt(process.argv[2] || '30')
+const NUMBER_OF_OFFERS = parseInt(process.argv[3] || NUMBER_OF_LISTINGS / 2)
+
+const NAMES = [
+  'Stan',
+  'Micah',
+  'Tyler',
+  'Josh',
+  'Daniel',
+  'Matthew',
+  'Kay',
+  'Пётр',
+  '계룡/鷄龍',
+  'Urist'
+]
+const CATEGORIES = ['Housing', 'Collectibles', 'For Sale', 'Services']
+const wordsFilePath = path.join(__dirname, "sample-words.json")
+const WORDLIST = JSON.parse(fs.readFileSync(wordsFilePath))
```

Comment:
You can actually just import the json file.

```js
import WORDLIST from './sample-words.json'
```

Reply:
I don't think node natively supports imports without special flags. I've switched it to require the words list file though.

Reply:
We're already doing it. https://github.com/OriginProtocol/origin-js/blob/master/src/resources/users.js#L3